keyvault/data-plane/azure-keyvault-extensions/pom.xml

```diff
@@ -55,11 +55,15 @@
     </dependency>
 
     <!-- Other Microsoft Azure Dependencies -->
+<!--    <dependency>-->
+<!--      <groupId>com.microsoft.azure</groupId>-->
+<!--      <artifactId>azure-mgmt-resources</artifactId>-->
+<!--    </dependency>-->
```

Comment:
Just delete the old code, don't comment it out - in all cases it shouldn't be kept.